app/styles/component/_post-search-results-item.scss

```diff
@@ -0,0 +1,49 @@
+$post-search-results-item-image-height: 48px;
+
+.post-search-results-item {
+  margin: 18px 0;
+  min-height: $post-search-results-item-image-height;
+
+  // Clearfix
+  &::after {
+    clear: both;
+    content: "";
+    display: block;
+    height: 0;
+    visibility: hidden;
+  }
+
+  &__title {
+    color: $wds-fandom-color-link;
+    display: inherit;
+    font-size: $wds-font-size-xs;
+    font-weight: $wds-font-weight-medium;
+    letter-spacing: .03px;
+    line-height: 1;
+    margin-bottom: 9px;
+  }
+
+  &__details {
+    align-items: center;
+    color: $wds-color-dark-gray;
+    display: flex;
+    fill: currentColor;
```

Comment:
I think it's not needed here. DS `.wds-icon` class already have it.

Reply:
Fixed in 1546f68